iOSClient/Utility/NCContentPresenter.swift

```diff
@@ -60,63 +60,51 @@ class NCContentPresenter: NSObject {
         case info
     }
     
-    @objc private var lastErrorCode: Int = 0
-
     //MARK: - Message
     
-    @objc func messageNotification(_ title: String, description: String?, delay: TimeInterval, type: messageType, errorCode: Int, forced: Bool = false) {
+    @objc func messageNotification(_ title: String, description: String?, delay: TimeInterval, type: messageType, errorCode: Int) {
+        messageNotification(title, description: description, delay: delay, type: type, errorCode: errorCode, priority: .normal, dropEnqueuedEntries: false)
+    }
+    
+    func messageNotification(_ title: String, description: String?, delay: TimeInterval, type: messageType, errorCode: Int, priority: EKAttributes.Precedence.Priority = .normal, dropEnqueuedEntries: Bool = false) {
```

Comment:
I still don't see the use of this parameter. It is not used in the function. If it is needed for future reference please add it as documentation.
```suggestion
    func messageNotification(_ title: String, description: String?, delay: TimeInterval, type: messageType, errorCode: Int, dropEnqueuedEntries: Bool = false) {
```

Reply:
https://github.com/nextcloud/ios/blob/3508a96adf50aa30f2f9c5c8998cd05eb7a18b4a/iOSClient/Networking/NCNetworkingE2EE.swift#L262

Reply:
have you look the callers in this PR ? or I didn't understand :D

Reply:
Thanks for the example, but that is a) not the `messageNotification` function and b) the caller, not the implementation.
Please check the [entire function](https://github.com/nextcloud/ios/blob/3508a96adf50aa30f2f9c5c8998cd05eb7a18b4a/iOSClient/Utility/NCContentPresenter.swift#L69-L89) again. The `priority` parameter is not used.

Reply:
uuu sorry my error

Reply:
ja ja ... my error, I Correct.

Reply:
https://github.com/nextcloud/ios/blob/3508a96adf50aa30f2f9c5c8998cd05eb7a18b4a/iOSClient/Networking/NCNetworking.swift#L208

Reply:
Yes, but this is the same as:
```swift
NCContentPresenter.shared.messageNotification("_error_", description: "_error_creation_file_", [...], priority: .normal)
```
There's no difference in the implementation and execution.

Reply:
very strange I remember putting it on ... look now